demisto_sdk/commands/content_graph/README.md

```diff
@@ -0,0 +1,73 @@
+## Content Graph Commands
+
+**A set of commands used for creating, loading and managing a graph database representation of content repository.**
+
+### Architecture
+The database is implemented with [neo4j](https://neo4j.com/) platform, and populated with data using [neo4j python driver](https://neo4j.com/docs/api/python-driver/current/api.html).
+In the database, every content object has a unique **node** which contains its properties. Nodes of content objects that are associated with each other (E.g., a playbook A uses a script B) have a directed **relationship** between them.
+
+![Architecture](images/architecture.png)
+
+#### Relationship Types
```

Comment:
What's the order of this list?

Reply:
it's an enum, the order is defined in the class (why does it matter?)